starcheck/src/starcheck.pl

```diff
@@ -440,8 +440,7 @@
 		$obs{$oflsid}->add_guide_summ($oflsid, \%guidesumm);
     }
     else {
-	my $obsid = $obs{$oflsid}->{obsid};
```

Comment:
I probably should have left this line (the print warning statement is likely going to also spit out an "undefined $obsid" warning.  But I'll fix it in a future PR).